agent/test-agent-cli/examples/example_test_agent_cli/example-test.sh

```diff
@@ -0,0 +1,70 @@
+# Get the configuration details for test and save it in a JSON file
+# Set the Task_state as running(This can be checked in Kubectl describe test or testsys status)
+test-agent-cli init > ExampleConfig.json
+
+echo "starting the Example test"
+
+#Use JQ to read the config JSON
+#https://www.linode.com/docs/guides/using-jq-to-process-json-on-the-command-line/
+person=$(jq '.person' ExampleConfig.json)
+helloCount=$(jq '.helloCount' ExampleConfig.json)
+helloDurationMilliseconds=$(jq '.helloDurationMilliseconds' ExampleConfig.json)
+passed=0
+
+echo "$person"
+echo "$helloDurationMilliseconds"
+echo "$helloCount"
+
+
+# Get the number of retries allowed in case of a failing test
+retries=$(test-agent-cli retry-count)
+echo "no of retires are $retries"
+
+#Perform the testing using bash scripts.
+test () {
+    for ((i = 1; i <= helloCount; i++))
+    do
+        echo "Hello ${i} to ${person}"
+        sleep "${helloDurationMilliseconds}"
+    done
+}
+
+execute () {
+    test
+    if [ $? -eq 0 ]
+    then
+        passed=1
+    fi
+}
+
+execute
+
+if [ $passed -eq 0 ] && [ $retries -gt 0 ]
+then
+    for ((i = 1; i <= $retries; i ++))
+    do
+        echo "Test failed again, retrying ${i} of $retries"
+        execute
+        # This will save the result of current iteration in a directory on
+        # kubernetes cluster, that will contain the info how many test cases
+        # failed, passed, ignored
+        # Save the result for every retry
+        test-agent-cli send-result --o Pass --num-passed 1 --num-failed 0 --num-skipped 0
+        if [ $passed -eq 1 ]
+        then
+            break
+        fi
```

Comment:
```suggestion
        echo "Test failed again, retrying ${i} of $retries"
        # This will save the result of current iteration in a directory on
        # kubernetes cluster, that will contain the info how many test cases
        # failed, passed, ignored
        # Save the result for every retry
        test-agent-cli send-result --o Pass --num-passed 1 --num-failed 0 --num-skipped 0
       
        if test; then
            break
        fi
```